### PR TITLE
check_expired_locked_rules, check_deletable_replicas and check_ok_rules probes

### PR DESCRIPTION
**Pull request by FernandoGarzon**

### CODE REVIEW

File: common/check_deletable_replicas [hunk 1]
```diff
@@ -8,70 +8,60 @@
...
 '''
-Probe to check the queues of messages to submit by Hermes to the broker
+Probe to check the number of deletable files per rse, as well as
+the total deletable space per rse.
 '''
-
 from __future__ import print_function
 import sys
-
+from rucio.db.sqla.constants import ReplicaState
```
Comment (ericvaandering) [inline common/check_deletable_replicas:19]: What editor are you using? PyCharm has a very nice function called "organize imports" which groups things together, etc. If you don't have it don't worry, I will run this through it.

File: common/check_deletable_replicas [hunk 2]
```diff
@@ -8,70 +8,60 @@
...
+from sqlalchemy import func
 
 # Exit statuses
 OK, WARNING, CRITICAL, UNKNOWN = 0, 1, 2, 3
 
+PROM_SERVERS = config_get('monitor', 'prometheus_servers', raise_exception=False, default='')
+if PROM_SERVERS != '':
+    PROM_SERVERS = PROM_SERVERS.split(',')
 
-if __name__ == "__main__":
```
Comment (ericvaandering) [inline common/check_deletable_replicas:25]: Please don't change this. If the original used a main() then keep it. If it didn't don't worry.

File: common/check_deletable_replicas [hunk 3]
```diff
@@ -8,70 +8,60 @@
...
-                WHEN NOT MATCHED THEN INSERT(rse_id, source, used, files, updated_at, created_at)
-                VALUES (u.rse_id, 'expired', u.bytes, u.files, u.updated_at, u.updated_at);
+        registry = CollectorRegistry()
+        session = get_session()
+        a = session.query(models.RSE.rse.label('Node'),func.count(models.RSEFileAssociation.name).label('Files'), func.sum(models.RSEFileAssociation.bytes).label('bytes')).\
+        filter( or_(models.RSEFileAssociation.state != ReplicaState.BAD,
+            models.RSEFileAssociation.state!=ReplicaState.UNAVAILABLE,
+            models.RSEFileAssociation.state!=ReplicaState.TEMPORARY_UNAVAILABLE,
+            models.RSEFileAssociation.tombstone < datetime.utcnow())).join(models.RSEFileAssociation.rse).group_by(models.RSE.rse)
+        print(a)
```
Comment (ericvaandering) [inline common/check_deletable_replicas]: This should be removed in the final version

File: common/check_expired_locked_rules [hunk 4]
```diff
@@ -6,7 +6,7 @@
 # You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
 #
 # Authors:
-# - Cedric Serfon, <cedric.serfon@cern.ch>, 2015
+# - Cedric Serfon, <cedric.serfon@cern.ch>, 2015. Eric Vaandering, <ewv@fnal.gov> and Fernando Garzon, <oscar.fernando.garzon.miguez@cern.ch>
```
Comment (ericvaandering) [inline common/check_expired_locked_rules]: Put each of us on a separate line with 2020 please

File: common/check_expired_locked_rules [hunk 5]
```diff
@@ -67,24 +68,36 @@ def main():
...
+        registry = CollectorRegistry()
+        sub_query = session.query(func.count(models.ReplicationRule.id),models.ReplicationRule.rse_expression).\
+            join(models.DataIdentifier, (models.ReplicationRule.scope == models.DataIdentifier.scope) 
+            & (models.ReplicationRule.name == models.DataIdentifier.name)).\
+                filter(models.ReplicationRule.locked == True).\
+                    filter(models.DataIdentifier.expired_at != None).\
+                    filter(models.DataIdentifier.expired_at < datetime.utcnow()).\
+                        group_by(models.ReplicationRule.rse_expression)
+        dids = sub_query.all()
+        print(dids)
```
Comment (ericvaandering) [inline common/check_expired_locked_rules]: Not sure either of these print statements add anything.

File: common/check_ok_rules [hunk 6]
```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
```
Comment (ericvaandering) [inline common/check_ok_rules:1]: In my version, check_not_OK_rules is now part of check_rule_counts. Does it make sense to move this in too?

**Review by ericvaandering — CHANGES_REQUESTED** [review]

File: common/check_deletable_replicas (hunk 2 above)
Comment (FernandoGarzon) [reply common/check_deletable_replicas:25]: Set back as original

File: common/check_deletable_replicas (hunk 1 above)
Comment (FernandoGarzon) [reply common/check_deletable_replicas:19]: I'm using visual studio

File: common/check_ok_rules (hunk 6 above)
Comment (FernandoGarzon) [reply common/check_ok_rules:1]: Ok. Let me know if this needs to be removed.

**Pull request merged**